.gitignore

```
@@ -0,0 +1,15 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
+/.idea/assetWizardSettings.xml
```

I would rather ignore all Idea files. It likes to do a lot of changes there periodically. No need to store them in VCS  
```suggestion
.idea
```